anilist.py

```diff
@@ -832,7 +819,7 @@ def map_watchcount_to_seasons(title: str, season_mappings: List[AnilistCustomMap
 
     for mapping in season_mappings:
         if watched_episodes >= mapping.start:
-            episodes_in_season = (watched_episodes - mapping.start + 1)
+            episodes_in_season = (watched_episodes - mapping.start)
```

Comment:
If we have a custom mapping where Anilist season 2 begins at Plex episode 14, the mapping would have `start: 14` in it.

If the user watches episode 14, `episodes_in_season` would be set to `0` and then nothing gets updated.

Please revert this change and the one in custom_mappings.py.